src/components/ModalFlows/CreateProposalScreens/ActionFees.js

```diff
@@ -34,93 +34,93 @@ function ActionFeesModal({ agreement, onCreateTransaction }) {
   return (
     <div>
       <Field
-        label="Transaction fees"
         css={`
           color: ${theme.surfaceContentSecondary};
         `}
       >
         <span
           css={`
-            ${textStyle('body2')}
+            ${compactMode ? textStyle('body3') : textStyle('body2')}
```

Comment:
we usually don't change the text size when compact, did you see it too big on mobile? ah also we need to add the `;` at the end of the `textStyle('body2');`

Reply:
For this specific case it was looking differente than the desing, but i can leave it as you use and in case you feel is bigger, i'll back to this.